wire-ios/Wire-iOS/Sources/Analytics/Analytics.swift

```diff
@@ -30,7 +30,7 @@ final class Analytics: NSObject {
     static var shared: Analytics!
 
     required init(optedOut: Bool) {
```

Comment:
I guess you can now also remove the `optedOut` property and therefore maybe also make `shared` a `let` and non optional:

```swift
static let shared = Analytics()
```

They maybe some subtleties to check around this such as does the singleton need to be created at a certain time.

Reply:
I removed the optedOut from the init. https://github.com/wireapp/wire-ios/commit/275400698c90736fa29d4661b62d47d01c5c26bc

I'm planning to completely get rid of `Analytics.swift` but on a separate PR.